packages/at_auth/lib/at_auth.dart

```diff
@@ -1,18 +1,26 @@
 library at_auth;
```

Comment:
I'd like to see documentation in here which explains how this package is used or at least tells the user which of the many files being exported are the most important